### PR TITLE
Add images and portfolio Content

### PR DESCRIPTION
**Pull request by Franky237**
- Created a Repository
- Added images
- Added content with Html
- Styling with CSS
- Setting up linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,47 @@
...
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
-
-### Setup
+- Internet
+- Browsers
```
Comment (Lizdev-05) [inline README.md]: -Kindly make your READme more descriptive by adding the necessary information that will help anyone that might want to use your code. 👇
readme

File: index.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Mobile Version</title>
+    <script src="https://kit.fontawesome.com/2af886883b.js" crossorigin="anonymous"></script>
+</head>
+<link rel="stylesheet" href="style.css">
+<body>
+    <section>
+        <div class="image1">
+        <div class="img-1">
+        <img  src="assets/img/Snapshoot Portfolio (1).jpg">
```
Comment (Lizdev-05) [inline index.html]: - Kindly, check the image path, the image is broken. 
- You can remove the whitespace between the image path and make it look like this 👉
img
- **Note:** You will have to rename it also. 😊
- You can rename it to any name of yours, just make sure there is no space between them. 
- This is what you have presently...👇
no-img

File: index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        </div>
+        <div>
+            <button type="button" class="button-1">See Project</button>
+        </div>
+    </div>
+    </section>
+    <section>
+        <div class="image1">
+        <div class="img-1">
+        <img src="assets/img/Snapshoot Portfolio (2).jpg">
```
Comment (Lizdev-05) [inline index.html]: Kindly change the image path of this also.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+        </div>
+        <div>
+            <button type="button" class="button-1">See Project</button>
+        </div>
+    </div>
+    </section>
+    <section>
+        <div class="image1">
+        <div class="img-1">   
+        <img src="assets/img/Snapshoot Portfolio (3).jpg">
```
Comment (Lizdev-05) [inline index.html]: Kindly change the image path of this also.☺

File: index.html [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+        </div>
+        <div>
+            <button type="button" class="button-1">See Project</button>
+        </div>
+    </div>
+    </section>
+    <section>
+        <div class="image1">
+        <div class="img-1">   
+        <img src="assets/img/Snapshoot Portfolio (4).jpg">
```
Comment (Lizdev-05) [inline index.html]: Kindly change the image path of this also.☺

File: index.html [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+                </div>
+                <div class="lang1">
+                    <li><img src="Assets/img/Ellipse 1 (1).jpg"></li>
+                    <li class="java">HTML</li>
+                </div>
+                <div class="lang1">
+                    <li><img src="Assets/img/Ellipse 1 (2).jpg"></li>
+                    <li class="java">CSS</li>
+                </div>
+                </ul>
```
Comment (Lizdev-05) [inline index.html]: Kindly rename the image paths. Make sure there is no space whatsoever ☺

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @Franky237, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there-reach

## Highlights
- Correct GitHub Flow 👍🏾
- Good commit history 🥇
- Good PR title/description 💯
- All linters passed 🥇

## Required Changes ♻️
- Kindly add the previous project, 'Setup and mobile first' to this code base. It is a continuation and not a divisive project. 😉
- Kindly make the icons to be bolder  as we have them in the [template]☺(https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- Kindly, align these items 👉 
align


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @Franky237, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**